plugins/module_utils/network/eos/config/acls/constants.py

```diff
@@ -0,0 +1,191 @@
+# constants.py
+
+"""This module defines project-level constants."""
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+PORT_NUMBERS = [
```

Comment:
Add a docstring below the list with a description. It is not clear what this list covers.